crates/aptos/src/move_tool/mod.rs

```diff
@@ -594,6 +600,97 @@ impl CliCommand<TransactionSummary> for PublishPackage {
     }
 }
 
+/// Create a resource account using the current account and the seed phrase
+#[derive(Parser)]
+pub struct CreateResourceAccountAddress {
+    #[clap(long)]
+    pub(crate) account_address: AccountAddress,
+
+    #[clap(long)]
+    pub(crate) seed: String,
+
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<()> for CreateResourceAccountAddress {
+    fn command_name(&self) -> &'static str {
+        "CreateResourceAccountAddress"
+    }
+
+    async fn execute(self) -> CliTypedResult<()> {
+        let resource_address =
+            create_resource_address(self.account_address, &bcs::to_bytes(&self.seed).unwrap());
+        println!("Resource account address created `{}`", resource_address);
+        Ok(())
+    }
+}
+
+/// Publishes the modules in a Move package to the Aptos blockchain under a resource account
+#[derive(Parser)]
+pub struct CreateResourceAccountAndPublishPackage {
+    #[clap(long)]
+    pub(crate) seed: String,
+
+    /// Whether to override the check for maximal size of published data
+    #[clap(long)]
+    pub(crate) override_size_check: bool,
+
+    #[clap(flatten)]
+    pub(crate) included_artifacts_args: IncludedArtifactsArgs,
+    #[clap(flatten)]
+    pub(crate) move_options: MovePackageDir,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for CreateResourceAccountAndPublishPackage {
```

Comment:
It might be good to ask users to confirm that they're publishing to the resource account address (displayed in the prompt). Example: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/src/stake/mod.rs#L486

Reply:
good point! just updated : )